fbw-a380x/src/wasm/systems/a380_systems/src/air_conditioning/cpiom_b.rs

```diff
@@ -363,15 +361,15 @@ impl AirGenerationSystemApplication {
     fn absolute_flow_calculation(
         &self,
         acs_overhead: &impl AirConditioningOverheadShared,
-        number_of_passengers: usize,
         pack: Pack,
         pressurization: &impl CabinAltitude,
     ) -> MassRate {
         let passenger_factor: f64 = if matches!(
             acs_overhead.flow_selector_position(),
             OverheadFlowSelector::Norm
         ) {
-            number_of_passengers as f64 / Self::A380_TOTAL_NUMBER_OF_PASSENGERS
+            // Minimum 40% flow to maintain temperature with zero pax
+            (self.pax_number_fms as f64 / Self::A380_PASSENGER_FACTOR).max(0.4)
```

Comment:
Shouldn't that be like this?
```suggestion
            (self.pax_number_fms as f64 / Self::A380_PASSENGER_FACTOR).clamp(0.4, 1.0)
```

Reply:
No, I intentionally made it so it can go above 100% flow when the aircraft is full (for reference, when it's in manual, the selection is between 80% - 100% -120%).